### PR TITLE
Crossbrowsing implemented

### PR DESCRIPTION
**Pull request by estebanpalomares**
safari test cases, will be execued in a macbook

### CODE REVIEW

**Review by wiensamo — APPROVED** [review]

File: src/main/java/com/gworks/ui/controller/EdgeWebDriverController.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.gworks.ui.controller;
+
+import com.gworks.ui.services.EdgeWebDriverService;
+import org.openqa.selenium.WebDriver;
+
+public class EdgeWebDriverController {
+
+    private final EdgeWebDriverService edgeWebDriverService = new EdgeWebDriverService();
```
Comment (luz-angelica) [inline src/main/java/com/gworks/ui/controller/EdgeWebDriverController.java:8]: I would like to add an _ (underscore) before the variable when we are creating an instance.
Just to make it neater.

File: src/test/java/com/gworks/stepDefinitions/ui/config/UiListener.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import io.qameta.allure.Attachment;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.testng.ITestListener;
+import org.testng.ITestResult;
+
+import java.io.ByteArrayInputStream;
+
+public class UiListener implements ITestListener {
```
Comment (luz-angelica) [inline src/test/java/com/gworks/stepDefinitions/ui/config/UiListener.java:13]: is this class related to these changes?

**Review by luz-angelica — APPROVED** [review]
Good Job!

File: src/test/java/com/gworks/stepDefinitions/ui/config/UiListener.java (hunk 2 above)
Comment (estebanpalomares) [reply src/test/java/com/gworks/stepDefinitions/ui/config/UiListener.java:13]: nope
